src/components/TokenBottomSheet.tsx

```diff
@@ -271,6 +271,7 @@ function TokenBottomSheet({
         keyExtractor={(item) => item.tokenId}
         contentContainerStyle={{
           paddingBottom: insets.bottom,
+          backgroundColor: Colors.backgroundPrimary,
```

Comment:
guess adding one here doesn't hurt anything, but is this required if we specify one in the Navigator?

Reply:
oh yes this is needed for when this TokenBottomSheet is used as a normal component (the swap screen)